xbmc/cores/VideoPlayer/VideoRenderers/LinuxRendererGLES.cpp

```diff
@@ -46,6 +46,7 @@ CLinuxRendererGLES::~CLinuxRendererGLES()
   UnInit();
 
   ReleaseShaders();
+  free(m_planeBuffer), m_planeBuffer = nullptr;
```

Comment:
```suggestion
  free(m_planeBuffer);
  m_planeBuffer = nullptr;
```
Code style: One statement per line